.cypress/integration/2_notebooks.spec.js

```diff
@@ -177,55 +146,46 @@ describe('Testing notebooks table', () => {
 
     // keep a notebook for testing
     cy.get('.euiButton__text').contains('Create notebook').click();
-    cy.wait(delay);
     cy.get('input.euiFieldText').type(TEST_NOTEBOOK);
     cy.get('.euiButton__text')
       .contains(/^Create$/)
       .click();
-    cy.wait(delay * 2);
   });
 });
 
 describe('Test reporting integration if plugin installed', () => {
   beforeEach(() => {
```

Comment:
<img width="679" alt="Screenshot 2023-05-19 at 9 43 58 AM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/f44e4f32-a710-4658-a9fc-d9f215081c19">
Installed reporting integration plugin and running "yarn osd bootsrap" encountered the above error. Adjusting the package.json file allowed it to run successfully.

Reply:
https://github.com/opensearch-project/dashboards-reporting/blob/main/package.json